recipe/meta.yaml

```diff
@@ -126,3 +123,7 @@ extra:
     - jezdez
     - kenodegard
     - travishathaway
+    - beeankha
```

Comment:
Not sure if that is needed at the moment, @beeankha unless you want to be a package reviewer?

Reply:
I went ahead and removed my name from the list